### ISSUE TITLE
demo in the README.md

### ISSUE THREAD

**Issue by gpgabriel** [KanoComputing/x-carousel#7]
Not exactly sure about this one but it would be great to have a small demo in the description.

**Comment by paulvarache** [comment]
Agreed. A nice addition to the link to the docs/demo gh-pages for a quick look on how to use it
